src/main/java/org/fsg1/fmms/backend/resources/AppBinder.java

```diff
@@ -0,0 +1,24 @@
+package org.fsg1.fmms.backend.resources;
+
+import org.fsg1.fmms.backend.database.Connection;
+import org.glassfish.jersey.internal.inject.AbstractBinder;
+import org.fsg1.fmms.backend.services.CurriculumService;
+
+import javax.inject.Singleton;
+
+/**
+ * The AppBinder. This resolves dependencies using Jersey's HK2 DI.
+ */
+public final class AppBinder extends AbstractBinder {
+
+    /**
+     * Configures dependencies. Dependencies can be shared in a Singleton, recreated for every request
+     * and more. Read HK2's DI documentation.
+     */
+    @Override
+    protected void configure() {
+        //Connection conn = new Connection();
+        bind(Connection.class).to(Connection.class).in(Singleton.class);
```

Comment:
the to.() is not really needed

Reply:
It is. Dependency injection will break if you leave it out.